packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -68,13 +68,14 @@ const InputGroup = ({ children, className }) => {
     );
 };
 
+const autocomplete_value = 'none';
 const TaxResidenceSelect = ({ field, errors, setFieldValue, values, is_changeable, residence_list }) => (
     <React.Fragment>
         <DesktopWrapper>
             <Autocomplete
                 {...field}
                 data-lpignore='true'
-                autoComplete='new-password' // prevent chrome autocomplete
+                autoComplete={autocomplete_value} // prevent chrome autocomplete
```

Comment:
@yaswanth-deriv can't we just pass `none` as the value here instead of having a variable for it?

Reply:
for some of the browsers, none was not working. Instead of none we may need to use some keyword.That is why i have used variable.